src/osemgrep/networking/Semgrep_App.ml

```diff
@@ -19,6 +19,7 @@ module Http_helpers = Http_helpers.Make (Lwt_platform)
 (* LATER: declared this in semgrep_output_v1.atd instead? *)
 type scan_id = string
 type app_block_override = string (* reason *) option
+type pro_engine_arch = Osx_arm64 | Osx_x86 | Manylinux
```

Comment:
meh. At least Manylinux_x86_64?

Also should be Osx_x86_64

Reply:
or Osx_x64, but I think the standard is x86_64.
x86 I think is used for 32bits x86.

Reply:
or maybe the standard is amd64. To study.